test/system/projects/samples_test.rb

```diff
@@ -759,7 +759,7 @@ class SamplesTest < ApplicationSystemTestCase
         fill_in I18n.t('projects.samples.attachments.concatenations.modal.basename'), with: 'concatenated file'
         check 'Delete originals'
         click_on I18n.t('projects.samples.attachments.concatenations.modal.submit_button')
-        !assert_html5_inputs_valid # rubocop:disable Lint/Void
+        assert_html5_inputs_valid
```

Comment:
This line can actually be removed. We check for if the html 5 inputs are valid a few lines below

Reply:
I'm not sure what happened here. If I accidentally deleted something or my IDE auto formatted it. I did not intend to touch this test. 9abbbbd57c12534cbe3fbab885b825516ddf48c0.

Reply:
Yeah the IDE probably formatted it!